ecmascript/ast/src/jsdoc.rs

```diff
@@ -0,0 +1,322 @@
+use crate::{Expr, Ident, Str, TsEntityName, TsTypeParamDecl};
+use swc_common::{ast_node, Span};
+
+#[ast_node]
+pub struct JsDoc {
+    pub span: Span,
+
+    pub description: Str,
+    pub tags: Vec<JsDocTagItem>,
+}
+
+/// JsDoc tag with comments.
+#[ast_node]
+pub struct JsDocTagItem {
```

Comment:
What about just `JsDocTag` to copy what's done in the TS compiler (and it's shorter 🙂)?

![image](https://user-images.githubusercontent.com/1609021/84459781-455aa000-ac36-11ea-8544-f95909b39c7d.png)

Reply:
Oh, I should scroll down to see the enum. Sorry! That is awesome you've implemented all those!!